pkg/splunk/enterprise/names.go

```diff
@@ -152,6 +158,11 @@ const (
 	livenessProbeTimeoutSec      = 30
 	livenessProbePeriodSec       = 30
 
+	// Startup probe time values
+	startupProbeDefaultDelaySec = 10
+	startupProbeTimeoutSec      = 5
+	startupProbePeriodSec       = 5
```

Comment:
These values are good. I think to avoid getting into any cyclic restarts caused by startup probes, particularly  on a pod reset for any lag on the Splunk side(Say, an already existing Splunk instance having large number of buckets, joining the cluster, destructive re-sync, etc...), i think it is safe to also add a large `FailureThreshold` value, just in the case of Startup Probe. Generally, all our pod resets take around 80 seconds(Cluster Manager is exception, but i guess should be OK to respond on mgmt port), so, considering the Period of 5 seconds, we can introduce the default `FailureThreshold` to 16 for the Startup Probe alone. We leave the default 3 for other probes. Thoughts?

CC: @vivekr-splunk